reactive/kubernetes_master.py

```diff
@@ -2971,6 +2977,11 @@ def _write_encryption_config():
     )
 
 
+@when_any('config.changed.pod-security-policy')
+def pod_security_policy_config_changed():
+    clear_flag('kubernetes-master.pod-security-policy.applied')
```

Comment:
I know, we're missing tests, that's in progress.
Have you tested this on a fresh deploy and not just an upgrade? I know config-changed triggers during install which can cause strange behavior. I'm not clear if clear_flag fails if the flag has never been set, it might be fine.

Reply:
Yes, tested on fresh new and upgrade. Both clear the flag kubernetes-master.pod-security-policy.applied and the policy gets re-applied.